HangmanGame/ViewController.swift

```diff
@@ -8,12 +8,165 @@
 import UIKit
 
 class ViewController: UIViewController {
-
+    
+    @IBOutlet weak var lblScore: UILabel!
+    @IBOutlet weak var correctWord: UILabel!
+    @IBOutlet weak var lblWrongGuess: UILabel!
+    @IBOutlet weak var imgHead: UIImageView!
+    @IBOutlet weak var imgBody: UIImageView!
+    @IBOutlet weak var imgLeftArm: UIImageView!
+    @IBOutlet weak var imgRightArm: UIImageView!
+    @IBOutlet weak var imgLeftLeg: UIImageView!
+    @IBOutlet weak var imgRightLeg: UIImageView!
+    @IBOutlet weak var btnGuess: UIButton!
+    @IBOutlet weak var textBox: UITextField!
+    
+    var score = 0 {
+        didSet {
+            lblScore.text = "Score \(score)"
+        }
+    }
+    
+    var wrongLetters = [String]()
+    var questionMark = [String]()
+    var words = ["MOOSEUM","ROADHOG","TRUNK","MILKSHAKE","BABOOM","BULLDOZER","HISSTORY"]
+    var selectedWord = ""
+    var letters = [Any]()
+    var numberOfLetters = 0
+    var wrongTempLetter = ""
+    var rightTempLetter = ""
+    var counter = 0
+    
     override func viewDidLoad() {
         super.viewDidLoad()
-        // Do any additional setup after loading the view.
+        words.shuffle()
+        loadLevel()
+        navigationItem.leftBarButtonItem = UIBarButtonItem(barButtonSystemItem: .refresh, target: self, action: #selector(loadLevel))
+        navigationController?.isToolbarHidden = false
+        
+    }
+    
+    private func divideLetters() {
+        selectedWord = words[1]
```

Comment:
Why index `1`?

Reply:
It's random. I could call 0. nothing special about it. 
I'll change to 0 to not generate any confusion.